project-api/project-api-test/src/test/java/io/ballerina/projects/test/TestBuildProject.java

```diff
@@ -438,7 +438,7 @@ public void testOverrideBuildOptions() {
 
         // Verify expected default buildOptions
         Assert.assertFalse(project.buildOptions().skipTests());
-        Assert.assertTrue(project.buildOptions().observabilityIncluded());
+        Assert.assertFalse(project.buildOptions().observabilityIncluded());
```

Comment:
We can revert this by reverting `project-api/project-api-test/src/test/resources/projectWithBuildOptions/Ballerina.toml`